src/Ethereum/RLP.cpp

```diff
@@ -140,137 +134,3 @@ Data RLP::putint(uint64_t i) noexcept {
     };
     // clang-format on
 }
-
-Data RLP::decodeRawTransaction(const Data& data) {
-    auto decoded = decode(data).decoded;
-    if (decoded.size() < 9) {
-        return {};
-    }
-    auto result = json {
-        {"nonce", hexEncoded(decoded[0])},
-        {"gasPrice", hexEncoded(decoded[1])},
-        {"gas", hexEncoded(decoded[2])},
-        {"to", hexEncoded(decoded[3])},
-        {"value", hexEncoded(decoded[4])},
-        {"input", hexEncoded(decoded[5])},
-        {"v", hexEncoded(decoded[6])},
-        {"r", hexEncoded(decoded[7])},
-        {"s", hexEncoded(decoded[8])},
-    }.dump();
-    return Data(result.begin(), result.end());
-}
-
-static RLP::DecodedItem decodeList(const Data& input) {
-    RLP::DecodedItem item;
-    auto remainder = input;
-    while(true) {
-        auto listItem = RLP::decode(remainder);
-        item.decoded.push_back(listItem.decoded[0]);
-        if (listItem.remainder.size() == 0) {
-            break;
-        } else {
-            remainder = listItem.remainder;
-        }
-    }
-    return item;
-}
-
-static uint64_t decodeLength(const Data& data) {
```

Comment:
I'd like to keep RLP basic type decoding (tx decoding can be removed), what do you think?

Reply:
I've put back the decodeList, decodeLenght, and decode(rest) methods.